pkg/api/api.go

```diff
@@ -8,6 +9,7 @@ import (
 	"github.com/oklog/ulid"
 )
 
+// Command is a TRC command.
```

Comment:
Good job 👍 adding comments to variables will make it more readable. But don't forget to add comments to the WebAPI.go file as well

Reply:
Everything exported by `pkg/webapi` is commented.